syntax = "proto3";

message LocationEvent {
  int32 user_id = 1;
  sint32 latitude = 2;
  sint32 longitude = 3;
  string creation_time = 4;
}

message Empty { }

service LocationService {
    rpc Create(LocationEvent) returns (LocationEvent);
    rpc Get(Empty) returns (LocationEvent);
}